src/metrics/mongoose_metrics.erl

```diff
@@ -48,6 +48,7 @@
               remove_host_type_metrics/1, get_report_interval/0,
               sample_metric/1]).
 
+-define(PREFIXES, {?MODULE, prefixes}).
```

Comment:
I think an atom mongoose_metrics_prefixes would be _marginally_ faster but still faster.
Unless to have a tuple is some kind of convention to follow.

Reply:
Yeah, might be faster, but for once I didn't think about that, I kinda assumed from somewhere we have that convention 🤷🏽 🤔